cmd/chart-repo/chart_repo.go

```diff
@@ -45,6 +45,9 @@ func init() {
 		cmd.Flags().String("mongo-url", "localhost", "MongoDB URL (see https://godoc.org/labix.org/v2/mgo#Dial for format)")
 		cmd.Flags().String("mongo-database", "charts", "MongoDB database")
 		cmd.Flags().String("mongo-user", "", "MongoDB user")
+		// see version.go
+		cmd.Flags().StringVarP(&application, "application", "", "", "Name of the application using chart-repo")
```

Comment:
I would suggest being more explicit about what this is for, e.g. `user-agent-comment` or `user-agent-bundle` and explain in the description that it can be used to provide more info in the user agent string when querying chart repository servers.